### DISCUSSION TITLE
Support for Magic Wallets

### DISCUSSION THREAD

**Discussion by mynamebrody** [rainbow-me/rainbowkit#316] [Ideas]
[Magic](https://magic.link/) allows you to onboard users new to crypto with just an email, phone number, or social login.

It would be nice to support a wallet solution that is more custodial like experience for some users. It's not truly custodial as a user can export their keys if they would like.

**Comment by nickbytes** [comment]
@mynamebrody I like this idea. I noticed that web3-react was also trying Magic as an experimental connector in their latest release. This might need to land in wagmi before it could be integrated into RainbowKit, but would definitely like to do this. Will add to the roadmap.

**Reply by mynamebrody** [reply]
Thank you for the reply! Happy to provide any additional feedback around this if you'd like. Is there anywhere I can follow along with a public roadmap of sorts at all?

**Reply by nickbytes** [reply]
We're working on public roadmap now, will release sometime in next week.

**Reply by kesar** [reply]
we got it working using https://github.com/EveripediaNetwork/wagmi-magic-connector feel free to use it <3

an example of integration w rainbowkit: https://github.com/Royal-lobster/Rainbowkit-Magic
